website/docs/quickstarts/bigquery-qs.md

```diff
@@ -88,25 +88,22 @@ In order to let dbt connect to your warehouse, you'll need to generate a keyfile
 4. Click **Upload a Service Account JSON File** in settings.
 5. Select the JSON file you downloaded in [Generate BigQuery credentials](#generate-bigquery-credentials) and dbt Cloud will fill in all the necessary fields.
 6. Click **Test Connection**. This verifies that dbt Cloud can access your BigQuery account.
-7. Click **Next** if the test succeeded. If it failed, you might need to go back and regenerate your BigQuery credentials.
+7. Click **Next** if the test succeeds. If it fails, you might need to go back and regenerate your BigQuery credentials.
 
 
 ## Set up a dbt Cloud managed repository 
 <Snippet path="tutorial-managed-repo" />
 
 
-## Initialize your dbt project​ and start developing
+## Initialize your dbt project
```

Comment:
Removing this from here because we are starting development next section onwards.
We are making this section straightforward to follow.